src/constants/versions.ts

```diff
@@ -1,6 +1,6 @@
 module.exports = {
-  ANDROID_VERSION: "1.31.3",
-  ANDROID_KOTLIN_VERSION: "0.15.3",
-  ANDROID_GEO_VERSION: "0.3.0",
-  ANDROID_SDK_VERSION: "2.40.0",
+  ANDROID_VERSION: "1.32.0",
+  ANDROID_KOTLIN_VERSION: "0.16.0",
+  ANDROID_GEO_VERSION: "0.3.1",
```

Comment:
The latest version of geo is 0.3.0

Reply:
Looks like v_0.3.1 was published in maven but the release tag was not. 
Released a tag with : https://github.com/aws-amplify/amplify-android/releases/tag/release-geo_v0.3.1